blockchain/tx_pool.go

```diff
@@ -870,7 +882,7 @@ func (pool *TxPool) add(tx *types.Transaction, local bool) (bool, error) {
 	// (2) remove an old Tx with the largest nonce from queue to make a room for a new Tx with missing nonce
 	// (3) discard a new Tx if the new Tx does not have a missing nonce
 	// (4) discard underpriced transactions
-	if uint64(pool.all.Count()) >= pool.config.ExecSlotsAll+pool.config.NonExecSlotsAll {
+	if uint64(pool.all.Slots()+numSlots(tx)) >= pool.config.ExecSlotsAll+pool.config.NonExecSlotsAll {
```

Comment:
I think the equality sign should be dropped.
If `pool.all.Slots()` is 4, `numSlots(tx)` is 4, and RHS is 8, then we don't need to discard txs, right?

Reply:
Thanks! Yes, it needs to be dropped. https://github.com/klaytn/klaytn/pull/1733/commits/9dc8ba1f4939fb4d289d5481222b16b5214a22a0